### ISSUE TITLE
Include installer

### ISSUE THREAD

**Issue by sancarn** [realrasengan/gpt4all-wrapper-js#3]
Would be nice if the package included installation of gpt4all as well as running

Realise this is easier said than done.

**Comment by liamvgallagher** [comment]
One click install: `iex (irm gpt4all.tb.ag)`
I don't manage the link so be sure to check the code before you run it in case it changes.

**Comment by sancarn** [comment]
> iex

Powershell I assume?

**Comment by liamvgallagher** [comment]
Yes. Just paste it into power shell and it should do everything for you, I
assume you are on windows. It would be tedious to implement an auto install
for the JS package because of the variety of operating systems out there
and other factors.

**Comment by sancarn** [comment]
Windows works for me yeah

> It would be tedious to implement an auto install for the JS package because of the variety of operating systems out there and other factors.

Yeah, I know it all too well from deploying my own applications. That said it would still be very useful :)

**Comment by ATMnou** [comment]
gpt4all.tb.ag is dead now?
